Rlog/Extension/ScheduleListView+.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ScheduleListView+.swift
+//  Rlog
+//
+//  Created by 정지혁 on 2022/10/20.
+//
+
+import SwiftUI
+
+extension ScheduleListView {
+    struct StatusPicker: View {
```

Comment:
[메모] 

```suggestion
    private struct StatusPicker: View {
```

private 를 적용해보니 ScheduleListView에서 사용하는것까지 제한이되네요 ,,

Reply:
넵 맞습니다 ㅠㅠ